Constitution.txt

```diff
@@ -1,490 +1,264 @@
+3: Representatives shall be apportioned among the several States which may be included within this Union, according to their respective Citizen Populations. The actual Enumeration shall be made within three Years after the first Meeting of the Congress of these United States, and within every subsequent Term of five Years, in such Manner as they shall by Law direct. The Number of Representatives shall not exceed one for every thirty thousand Citizens, nor shall it be less than one for every one hundred thousand Citizens, excepting that each State shall have at least one Representative.
 
 4: When vacancies happen in the Representation from any State, the Executive Authority thereof shall issue Writs of Election to fill such Vacancies.
 
-5: The House of Representatives shall chuse their Speaker and other Officers; and shall have the sole Power of Impeachment.
+5: The House of Representatives shall choose their Speaker and other Officers; and shall have the sole Power of Impeachment against officials of the Federal government.
 
 Section 3
-1: The Senate of the United States shall be composed of two Senators from each State, chosen by the Legislature thereof,3 for six Years; and each Senator shall have one Vote.
+1: The Senate of these United States shall be composed of three Senators from each State, chosen by the Legislature thereof; and each Senator shall have one Vote.
 
-2: Immediately after they shall be assembled in Consequence of the first Election, they shall be divided as equally as may be into three Classes. The Seats of the Senators of the first Class shall be vacated at the Expiration of the second Year, of the second Class at the Expiration of the fourth Year, and of the third Class at the Expiration of the sixth Year, so that one third may be chosen every second Year; and if Vacancies happen by Resignation, or otherwise, during the Recess of the Legislature of any State, the Executive thereof may make temporary Appointments until the next Meeting of the Legislature, which shall then fill such Vacancies.4
+2: Each time a State joins the Union, it shall send three Senators to the Congress, of whom one will serve a term of two Years, one a term of four, and one a standard term of six, as designated by the State at the time that it joins the Union. Upon later election, each Senator from the State will serve six Years. 
```

Comment:
There are multiple purposes for moving from two to three Senators per State. Some I will expand upon later. The rest are:

- When there is an odd number of states, it is less likely that the Vice President would need to cast a tie-breaking vote in the Senate.
- When there is a selection for Senate every two years, the States are always guaranteed a regular indicator of approval or disapproval, that can send signals to their other senators.